appcues/src/main/java/com/appcues/debugger/AppcuesDebuggerManager.kt

```diff
@@ -72,16 +68,15 @@ internal class AppcuesDebuggerManager(
         coroutineScope.launch {
             viewModel.uiState.collect { state -> onBackPressCallback.isEnabled = state is Expanded }
         }
-        addDebuggerView(activity, viewModel)
-        application.registerActivityLifecycleCallbacks(this)
-        setDebuggerBackPressCallback(activity)
+        addDebuggerView(viewModel)
+        contextWrapper.getApplication().registerActivityLifecycleCallbacks(this)
     }
 
     fun stop() {
         coroutineScope.coroutineContext.cancelChildren()
         removeDebuggerView()
         debuggerViewModel?.viewModelScope?.cancel() // stop the VM from listening to app activity
-        application.unregisterActivityLifecycleCallbacks(this)
+        contextWrapper.getApplication().registerActivityLifecycleCallbacks(this)
```

Comment:
this should be `unregisterActivityLifecycleCallbacks`

Reply:
thanks!